packages/cli/src/tools/packageManager.js

```diff
@@ -61,8 +67,34 @@ export function uninstall(packageNames: Array<string>, options?: Options) {
     : executeCommand('npm', ['uninstall', ...packageNames, '--save'], options);
 }
 
-export function installAll(options?: Options) {
-  return shouldUseYarn(options)
-    ? executeCommand('yarn', ['install'], options)
-    : executeCommand('npm', ['install'], options);
+export function installCocoaPods() {
+  return executeCommand('sudo', ['gem', 'install', 'cocoapods'], {
+    silent: false,
```

Comment:
I needed to do this in order to have the input for the user password since it's using `sudo`, didn't have time yet to check how to do it without having the messages of actually installing the dep, will research a little bit more later.

Reply:
Yeah, good idea!

Reply:
Can we first try without `sudo` and then fallback to use it? For users with custom Ruby (not bundled with the system) it's not necessary

Reply:
Yeah, I can put that in.

Reply:
Done.